### ISSUE TITLE
Problem removing table top or bottom rules with latex command in pandoc 2.19

### ISSUE THREAD

**Issue by gnpan** [jgm/pandoc#8223]
Pandoc 2.19, macOS 12.5

With previous pandoc versions, table top/mid/bottom rules could be removed by including appropriate latex commands under header-includes. The following example with pandoc 2.19 removes top and bottom rules but adds two parentheses (), as shown in attached pdf. Same with grid tables.

```
---
header-includes: 
  - \let\toprule\relax
  - \let\bottomrule\relax
---

| c1 | c2 | c3 |
|:---|:---|:---|
| r1 | a | b |
| r2 | c | d | 

```
[test.pdf](https://github.com/jgm/pandoc/files/9310685/test.pdf)
[test.md](https://github.com/jgm/pandoc/files/9310721/test.md)

**Comment by tarleb** [comment]
See #8001 for the reasons behind this change, but maybe we can make that work again: `\midrule` is always followed by either `\endhead` or `\endfirsthead`, so the parens can be omitted. Same for `\bottomrule`, which will always be followed by `\end{longtable}`. This leaves `\toprule`: replacing `\toprule` with `\toprule\relax` seems to have no adverse effects in my tests.

For the time being, you can use `\renewcommand{\toprule}[2]{}` etc. to remove the rules.

**Comment by jgm** [comment]
In this connection we should also consider
https://github.com/jgm/pandoc/issues/7421#issuecomment-882497287

Revisiting my decision to add the `()` instead of `\relax`:  based on tests with test/tables.native, it seems that the reason I did this was because with `\midrule\relax` and `\bottomrule\relax` we get extra vertical space beneath the midrule and the bottomrule. Does anyone understand why?  It seems that using `\relax` instead of `()` would perhaps solve both problems, but we don't want this vertical space.

**Comment by jgm** [comment]
> replacing `\toprule` with `\toprule\relax` seems to have no adverse effects in my tests.

When I tried this, I got extra white space in the headerless tables in test/tables.native. e.g.

<img width="496" alt="Screen Shot 2022-08-22 at 8 16 28 AM" src="https://user-images.githubusercontent.com/3044/185956898-fc153686-1a5f-40d8-b7a8-3b994523ea5e.png">

With the current setup we don't get that blank line after the top rule.

**Comment by tarleb** [comment]
Oh. I guess this issue should be closed then. I don't know why I missed that, or why I got a different result.

**Comment by jgm** [comment]
I'd like to keep this open until I can understand why the `\relax` doesn't work.  Maybe a TeX guru can help.

**Comment by tarleb** [comment]
I found my mistake: I was experimenting in a raw tex file and had removed the `\endhead` command. I get the blank line when I add it back in.

**Comment by jgm** [comment]
See also #8242.

**Comment by tarleb** [comment]
It seems like all questions around this issue have been answered. Closing.

**Comment by jgm** [comment]
I was thinking we should keep this open in order to explore the solution proposed here:

>  The simplest fix would be to replace it with `\noalign{}` which inserts empty vertical box between table rows, thus not affecting the cell content.

Unless I was misunderstanding and this isn't a potential solution?

**Comment by tarleb** [comment]
Oh, I had missed that!

**Comment by jpk3333** [comment]
I've been trying to implement zebra shading in a longtable following the discussion [here](https://github.com/jgm/pandoc/issues/7421#issuecomment-882497287). I managed to get something that looked good by adding this before the table in my markdown file:

```
\rowcolors{2}{}{gray!25}
\setlength{\tabcolsep}{0pt}
\apptocmd{\toprule}{\hiderowcolors}{}{}
\apptocmd{\endhead}{\showrowcolors}{}{}
\apptocmd{\endfirsthead}{\showrowcolors}{}{}
```
The only problem was that two parentheses "()" were added into the top of the table. I then found my way to this thread. Following the discussion here, I discovered that if I used pandoc to compile my markdown file to LaTeX, I could get rid of the parentheses by replacing the instance of "\toprule()" to "\toprule\noalign{}" in the LaTeX file.

I guess my question is: what (and where) can I include in my markdown file so that Pandoc produces a PDF without the parentheses? I mean, I guess I can compile to LaTeX and make that change manually, but it'd be great if there were another way. Thanks!
